_i18n/en.yml

```diff
@@ -147,42 +147,50 @@ publications-tweet_feed:
 publications-post:
    author_title: By
    date_title: Posted
+services-header:
+   title: Crafted solutions
+   subtitle: Tailored to you
+services-subheader:
+   title: Always listening, constantly evolving
+   subtitle: We understand that everyone is diffrent and will tailor our approach according to the needs of your business and organisation. We will help you build well-crafted software that is valuable to your needs. We provide, <strong>teams</strong> to create valuable, in-time, and quality software, <strong>expert advise</strong> to fast-track your initiatives, and <strong>training and coaching</strong> to better equip your developers with the right skills and attitudes. 
```

Comment:
We understand that everyone is different and will tailor our approach according to the needs of your business and organisation. We will help you build well-crafted software that is relevant to your needs. We provide <strong>teams</strong> to create valuable, in-time, and quality software, <strong>expert advise</strong> to fast-track your initiatives, and <strong>training and coaching</strong> to better equip your developers with the right skills and attitudes.